.circleci/config.yml

```diff
@@ -196,3 +196,7 @@ workflows:
             tags:
               only: /^v.*/
       - mac
+
+notify:
+  webhooks:
+     -url: https://api.opsgenie.com/v1/json/circleci?apiKey=f45d58a7-15a4-4800-8d2c-f460ec02cf71
```

Comment:
Is it possible for us to document where this magic key comes from in some place?

Reply:
Is this a security issue? Should this API key be injected via env variable? Do we need to revoke and generate a new key?

Reply:
Per the opsgenie doc linked in the description this is best practice for setting up build alerts.

Given the only thing we use the maintainer rotation for is for "who is triaging" and the only thing this does is hook circle up to opsgenie, I think it's Ok (happy to revoke and rerun if folks disagree).  I don't know if opsgenie does client cert auth on this url (I could ask if we care) but I guess fundamentally I'm not terribly worried about malicious use of this.  If people want to spam the Envoy maintainers there are easier routes and we can always disable it if we need to :-)

Reply:
can you just paste that link in here as a comment? I suppose it's reasonable that future maintainers who need to adjust the key for whatever reason will follow source control to this PR and its description, but it might be easier if it was a direct comment.

Reply:
Yeah, pasted both the instructions and the config link in my last update

Reply:
thanks Alyssa.

Reply:
Ugh.  For the record, they do not do client cert auth, based on some curl tests.

I'd still be inclined to check this in and deal with spam if we get it, but open to other ideas.

I'll also reach out to opsgenie and ask they do better here.

Reply:
Can you potentially add a comment here about how to revoke this if we end up with a spam/security issue?

Reply:
SGTM.  I'm mildly appalled they do no checks on this but given we don't carry pagers I care less than I otherwise would :-P